### PR TITLE
Colors refactoring: DimTextColor is now SecondaryTextColor

### PR DESCRIPTION
**Pull request by vancura**
Colors refactoring: `DimTextColor` is now `SecondaryTextColor`.

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/Styles.cs [hunk 1]
```diff
@@ -78,8 +79,6 @@ public static class Styles
 		public static Color PrimaryBackgroundColor { get; internal set; }
 		public static Color SecondaryBackgroundLighterColor { get; internal set; }
 		public static Color SecondaryBackgroundDarkerColor { get; internal set; }
-		public static Color DimTextColor { get; internal set; }
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/Styles.cs]: This is a public API break, rather make `DimTextColor` route into `SecondaryTextColor`

Comment (vancura) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/Styles.cs]: What about https://github.com/mono/monodevelop/pull/2912/commits/9ad1c65d58ac3918e30e07c34e39835f334e7937 please?

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/Styles.cs [hunk 2]
```diff
@@ -79,6 +79,13 @@ public static class Styles
 		public static Color PrimaryBackgroundColor { get; internal set; }
 		public static Color SecondaryBackgroundLighterColor { get; internal set; }
 		public static Color SecondaryBackgroundDarkerColor { get; internal set; }
+
+		[Obsolete ("Please use SecondaryTextColor")]
+		public static Color DimTextColor { get => SecondaryTextColor; }
+
+		[Obsolete ("Please use SecondaryTextColorHexString")]
+		public static string DimTextColorHexString { get => DimTextColorHexString; }
```
Comment (mrward) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/Styles.cs]: This is calling itself? Should be `get => SecondaryTextColorHexString`?

**Comment by vancura** [comment]
All the issues fixed. I hope.

**Pull request merged**